lib/tasks/test_security_helper.rb

```diff
@@ -12,6 +12,11 @@ def self.brakeman(format: "human")
     require Rails.root.join('lib/extensions/brakeman_excludes_patch')
     Brakeman::AppTree.prepend(BrakemanExcludesPatch)
 
+    # Brakeman's fingerprint check does not work properly with engines
+    require "brakeman/warning"
+    require Rails.root.join('lib/extensions/brakeman_fingerprint_patch')
+    Brakeman::Warning.prepend(BrakemanFingerprintPatch)
```

Comment:
@jrafanie For now, I'm just copying the patching pattern we had before.  I plan to have a separate PR where I do more of a "safe-patch" approach that checks the various methods, but I didn't want to introduce too much in this PR, especially if I want to backport it.